packages/sdk/src/Redemptions.ts

```diff
@@ -41,6 +43,8 @@ export class Redemptions {
 			payload = { customer }
 		}
 
-		return this.client.post(`/redemptions/${encode(redemptionId)}/rollback`, payload, { qs })
```

Comment:
Last param should not be inside `{}`, it should just be passed directly - aka `qs` instead of `{ qs }`. Currently this does not work.